lib/view_model/buy/buy_sell_view_model.dart

```diff
@@ -149,13 +149,12 @@ abstract class BuySellViewModelBase extends WalletChangeListenerViewModel with S
 
   @computed
   bool get isReadyToTrade {
-    final hasSelectedQuote = selectedQuote != null;
+    // final hasSelectedQuote = selectedQuote != null;
     final hasSelectedPaymentMethod = selectedPaymentMethod != null;
     final isPaymentMethodLoaded = paymentMethodState is PaymentMethodLoaded;
     final isBuySellQuotLoaded = buySellQuotState is BuySellQuotLoaded;
 
-    return hasSelectedQuote &&
-        hasSelectedPaymentMethod &&
+    return hasSelectedPaymentMethod &&
```

Comment:
why this change? it shouldn't be ready until the user selects a quote

Reply:
With this change we do not have any details regarding the quote because we don't select the quote by default from EUR-only providers (if the currency is SEPA enabled), as this would be unreliable. We just do not calculate anything in terms of price because
- provider cannot give use price in local currency
- we cannot calculate the price on our own because we do not include SEPA fees and provider fees, so using for example priceapi wouldn't work.

Yet we still need to mark the trade as ready to allow the user to click the button - so if there is any quote (even in foreign currency) we unlock the button